### ISSUE TITLE
Timestamps are not preserved in PNGs

### ISSUE THREAD

**Issue by eldelacajita** [Huluti/Curtail#205]
Hi! I love the fact that Curtail can usually preserve the file attributes like the timestamps. This allows me to optimize images in place while still being able to sort and search them correctly by original date.

However, I just noticed this isn't the case for PNGs. I have the "preserve file attributes if possible" option enabled, but the timestamp of all PNGs gets changed to the current time.

I'm using the Flatpak version of Curtail, v. 1.8.0 on EndeavourOS.

**Comment by Huluti** [comment]
Hello! Is it still the case with the latest version?

**Comment by eldelacajita** [comment]
I have now version 1.10.0 via Flathub (not sure if it's the latest), and the issue is still there.

**Comment by Huluti** [comment]
Sorry for the answer time. Do you compress lossless or lossy? And do you speak about creation time or modification time?

**Comment by Huluti** [comment]
It seems to come from the lossy compression of pngquant library: https://github.com/kornelski/pngquant/issues/360

**Comment by eldelacajita** [comment]
Yes, it happens always when using lossy compression, but also sometimes when using lossless compresion. Both creation and modification time get modified.
